cmd/nvidia-container-runtime-hook/container_config.go

```diff
@@ -16,11 +14,11 @@ import (
 )
 
 const (
-	envCUDAVersion          = "CUDA_VERSION"
-	envNVRequirePrefix      = "NVIDIA_REQUIRE_"
-	envNVRequireCUDA        = envNVRequirePrefix + "CUDA"
-	envNVDisableRequire     = "NVIDIA_DISABLE_REQUIRE"
-	envNVVisibleDevices     = "NVIDIA_VISIBLE_DEVICES"
+	envCUDAVersion      = "CUDA_VERSION"
+	envNVRequirePrefix  = "NVIDIA_REQUIRE_"
+	envNVRequireCUDA    = envNVRequirePrefix + "CUDA"
+	envNVDisableRequire = "NVIDIA_DISABLE_REQUIRE"
+
 	envNVMigConfigDevices   = "NVIDIA_MIG_CONFIG_DEVICES"
 	envNVMigMonitorDevices  = "NVIDIA_MIG_MONITOR_DEVICES"
 	envNVImexChannels       = "NVIDIA_IMEX_CHANNELS"
```

Comment:
If you are moving NVIDIA_VISIBLE_DEVICES should you also move NVIDIA_IMEX_CHANNELS?

That said -- I'M not a huge fan of splitting these up. Does it make sense to create a `consts` package (or soemthing less specific) that both the image package and this one can import to get access to these, so we can keep them all in the same place?

Reply:
Since this code is importing the `image` package anyway, do you have an objection to putting them there?

Reply:
I have reworked these constants. They should now all be in the `image` package.